pyproject.toml

```diff
@@ -69,7 +69,7 @@ pylint = "^3.0.3"
 matplotlib = "^3.7.0"
 tensorflow = { version = "^2.14.1,<2.16", markers = "sys_platform == 'linux'" }
 torch = "^2.1.1"
-qibojit = { git = "https://github.com/qiboteam/qibojit.git" }
+qibojit = { git = "https://github.com/qiboteam/qibojit.git" , branch = "ccz" }
```

Comment:
It would be probably wiser to merge the `ccz` branch into main in `qibojit`. I would avoid branches deps in the `pyproject.toml`.

Reply:
Of course. This was just to show that the tests are passing.